packages/@lwc/template-compiler/src/__tests__/fixtures/svg/filter/expected.js

```diff
@@ -1,4 +1,20 @@
-import { registerTemplate } from "lwc";
+import { parseFragment, registerTemplate } from "lwc";
+const $fragment1 = parseFragment`<feFlood x="25%" y="25%" width="50%" height="50%" flood-color="green" flood-opacity="0.75"${1}${2}>`;
```

Comment:
This will not work. The generated element will be created in the HTML namespace instead of the SVG namespace. 

```js
const res = document.createRange().createContextualFragment(
  `<feFlood x="25%" y="25%" width="50%" height="50%" flood-color="green" flood-opacity="0.75">`
).firstChild;

console.log(res.namespaceURI)
// Actual: http://www.w3.org/1999/xhtml
// Expected: http://www.w3.org/2000/svg
```

As currently implemented, it's impossible to make static certain elements only of an SVG fragment. The entire SVG has to be static.

An alternative approach would be to expose another method called `parseSVGFragment` for handling this case. This function is similar to `parseFragment` with the key difference that it would wrap the fragment inside an `<svg>` tag.

Reply:
nice catch! i went with the parseSVGFragment approach https://github.com/salesforce/lwc/pull/2781/commits/1685fcd73389b9b9edd260ea58cb12726ec43892